docs/elements.md

```diff
@@ -1305,12 +1305,12 @@ Attribute | Type | Default | Description
 
 #### Example implementations
 
-- [demoCodeEditorAutograded]
-- [demoCodeUploadAutograded]
-- [demoAutograderSquare]
-- [demoAutograderNumpy]
-- [demoAutograderPandas]
-- [demoAutograderPlots]
+- [demoCodeEditorAutograded](https://github.com/PrairieLearn/PrairieLearn/tree/master/exampleCourse/questions/demoCodeEditorAutograded)
```

Comment:
@nicknytko these should be placed at the end after: 

```md

# Deprecated

## pl-variable-score

<!-- Switch to using reference style links for elements -->

[demoCodeEditorAutograded]: https://github.com/PrairieLearn/PrairieLearn/tree/master/exampleCourse/questions/demoCodeEditorAutograded
```

Reply:
Ah, TIL this was a thing